src/agent/src/rpc.rs

```diff
@@ -1846,9 +1912,17 @@ pub async fn start(
     server_address: &str,
     init_mode: bool,
 ) -> Result<TtrpcServer> {
+    #[cfg(feature = "sealed-secret")]
+    let c = ttrpc::asynchronous::Client::connect(CDH_ADDR)?;
+    #[cfg(feature = "sealed-secret")]
+    let client = Some(sealed_secret_ttrpc_async::SealedSecretServiceClient::new(
+        c.clone(),
+    ));
```

Comment:
Can we combine these sentences into one `feaeture` flag? s.t.
```
    #[cfg(feature = "sealed-secret")]
    let client = {
        let c = ttrpc::asynchronous::Client::connect(CDH_ADDR)?;
        Some(sealed_secret_ttrpc_async::SealedSecretServiceClient::new(c))
    };
```

Reply:
fixed, thanks.